TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -474,5 +476,143 @@ private SymbolReference CreateTableOrAliasName(CodeElementsParser.TableOrAliasNa
+                }
+
+                if (alterSequenceClauseContext.minValueClause() != null &&
+                    alterSequenceClauseContext.minValueClause().numeric_constant != null)
+                {
+                    SetOption(alterSequenceClauseContext.minValueClause(), AlterSequenceClauseType.MinValue,
+                        c => minValue = new SqlConstant((Token)c.numeric_constant));
+                }
+
+                if (alterSequenceClauseContext.maxValueClause() != null &&
+                    alterSequenceClauseContext.maxValueClause().numeric_constant != null)
+                {
+                    SetOption(alterSequenceClauseContext.maxValueClause(), AlterSequenceClauseType.MaxValue,
+                        c => maxValue = new SqlConstant((Token)c.numeric_constant));
+                }
+
+                if (alterSequenceClauseContext.cycle() != null)
+                {
+                    SetOption(alterSequenceClauseContext.cycle(), AlterSequenceClauseType.Cycle,
+                        c => cycle = new SyntaxProperty<bool>(true, ParseTreeUtils.GetFirstToken(c)));
+                }
+
+                if (alterSequenceClauseContext.cacheClause() != null &&
+                    alterSequenceClauseContext.cacheClause().numeric_constant != null)
+                {
+                    SetOption(alterSequenceClauseContext.cacheClause(), AlterSequenceClauseType.Cache,
+                        c => cacheSize = new SqlConstant((Token)c.numeric_constant));
+                }
+
+                if (alterSequenceClauseContext.SQL_ORDER() != null)
+                {
+                    SetOption(alterSequenceClauseContext.SQL_ORDER(), AlterSequenceClauseType.Order,
+                        c => ordered = new SyntaxProperty<bool>(true, ParseTreeUtils.GetFirstToken(c)));
+                }
+
+                if (alterSequenceClauseContext.noClauses() != null)
+                {
+                    var noKeyword = alterSequenceClauseContext.noClauses().SQL_NO();
+                    if (alterSequenceClauseContext.noClauses().cycle() != null)
+                    {
+                        SetOption(noKeyword, AlterSequenceClauseType.Cycle,
+                            c => cycle = new SyntaxProperty<bool>(false,
+                                ParseTreeUtils.GetTokenFromTerminalNode(noKeyword)));
+                    }
+
+                    if (alterSequenceClauseContext.noClauses().SQL_ORDER() != null)
+                    {
+                        SetOption(noKeyword, AlterSequenceClauseType.Order,
+                            c => ordered = new SyntaxProperty<bool>(false,
+                                ParseTreeUtils.GetTokenFromTerminalNode(noKeyword)));
+                    }
+
+                    if (alterSequenceClauseContext.noClauses().cache() != null)
+                    {
+                        SetOption(noKeyword, AlterSequenceClauseType.Cache,c=> {});
```

Comment:
We should also store the information about nocache, no minvalue and no maxvalue.
Can you add these properties to AlterSequenceStatement:
```
public SyntaxProperty<bool> Cache { get; }
public SyntaxProperty<bool> MinValue { get; }
public SyntaxProperty<bool> MaxValue { get; }
```
and se them here here.

Reply:
Yes, here they would have to be set to false and you'll  have to set them to true in upper part of the method too.